### DISCUSSION TITLE
Different results with different models (stochastic vs dynamical)

### DISCUSSION THREAD

**Discussion by shaln** [theislab/scvelo#894] [Q&A] [answered]
Hello, 

I previously ran the stochastic model on my pathological sample, as I wasn't sure if the computer I was using could run the dynamical model. The scvelo documentation also suggest that the stochastic model was good enough if computational power was an issue. The results I had from this model matches our hypothesis and other lab findings. However, when I tried running the dynamical model on the exact same sample (out of curiosity), the results were completely different. I understand that the mathematics behind each model is different, most of which I don't completely understand I'm afraid, but I wasn't expecting the resulting velocities to be so drastically different. 

In situations like this, how does one know which result is 'correct'? Is the dynamical model always the recommended one? The stochastic model fits our hypothesis and experimental results, but would choosing it on that basis alone be biased if the dynamical model is the recommended one?

<img width="824" alt="image" src="https://user-images.githubusercontent.com/39313061/176206068-7a42fd55-6fd5-4409-b280-c8d20e5f85b6.png">

**Comment by WeilerP** [comment, marked as answer]
@shaln, please have a look at #216 or #462, for example.
